browser-testing.md

```diff
@@ -1021,6 +1044,29 @@ The `value` method gets the value of the element matching the given selector:
 $value = $page->value('input[name=email]');
 ```
 
+<a name="with-in-iframe"></a>
+
+### withinIframe
+
+The `withinIframe` method allows you to interact with elements inside an iframe:
+
+```php
+$page->withinIframe('.iframe-container', function (WebPage $page) {
```

Comment:
https://github.com/pestphp/pest-plugin-browser/compare/v4.0.3...4.x

this should be an AwaitingPage... Plus maybe add the import on top.